### DISCUSSION TITLE
想把1.x版本应用到生产环境，但是现在还在公测阶段，会有风险吗？

### DISCUSSION THREAD

**Discussion by chengfengfengwang** [jd-opensource/micro-app#767] [General]
> 在正式版中接入vite子应用需要关闭沙箱，这会导致很多问题，暂时不建议接入。
> 在1.0公测版中我们推出了为vite打造的沙箱方案，对于新接入的项目，推荐使用新的方案，具体详情请跳转[1.0公测版文档](https://micro-zoe.com/docs/1.x/#/zh-cn/start)。

在github文档看到这句话，而且我的主应用是vue cli 5.x，子应用是vite3.x，想知道我用1.x版本会有风险吗？

**Comment by yahoolcj** [comment]
有 setData变成了异步

**Reply by chengfengfengwang** [reply]
setData本身不就是异步的吗
